config/default.js

```diff
@@ -76,6 +76,7 @@ module.exports = {
   },
   stripe: {
     api_key: process.env.STRIPE_TEST_PUBLIC_KEY || null,
-    api_secret: process.env.STRIPE_TEST_SECRET_KEY || null
+    api_secret: process.env.STRIPE_TEST_SECRET_KEY || null,
+    enthusiast_plan_id: 'plan_Fc2mb0eKhaEzu5' // TODO DECIDE IF THIS SHOULD BE .ENV
```

Comment:
Oh, this is a good question. It seems the plan IDs are not meant to be secret since they have to be passed around in JavaScript, but I've also started coming around to the idea of putting more things in environment variables anyway because they'd be more flexible to deal with if we ever have more than one deployment. We could possibly check for an environment variable first and then leave the test plan ID in as a fallback.

Reply:
Also, since we haven't confirmed any plan names, let's rename this to something generic. I'm thinking "tier1" (see reasons above) but we've also thought about it as "level1" or "subscription1" — the key point being that they're generic identifiers (we'll nail down the appropriate "marketing labels" later) with a number that makes it clearer how the tiers are stacked.